### PR TITLE
[W4.11e][T12-2]Zhou Zegang

### PR DESCRIPTION
**Pull request by ZZG229**
Edit the Junit test

### CODE REVIEW

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -43,4 +43,28 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    private void assertHasNull(Object... objects) {assertTrue(Utils.isAnyNull(objects));}
```
Comment (eugenepeh) [inline test/java/seedu/addressbook/common/UtilsTest.java:47]: As mentioned in the tutorial, we usually don't recommend single line method.

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 2]
```diff
@@ -43,4 +43,28 @@ private void assertAreUnique(Object... objects) {
...
+
+        //only one object
+        assertHasNull((Object) null);
+        assertNoNull(1);
+        assertNoNull("");
+        assertNoNull("abc");
+
+        //contains no null object
+        assertNoNull("abc","a");
+        assertNoNull(1,Integer.valueOf(1));
```
Comment (eugenepeh) [inline test/java/seedu/addressbook/common/UtilsTest.java:63]: Good consistency within the code, however note that our coding standard requires space after the comma.

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 3]
```diff
@@ -43,4 +43,28 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    private void assertHasNull(Object... objects) {assertTrue(Utils.isAnyNull(objects));}
+
+    private void assertNoNull(Object... objects) {assertFalse(Utils.isAnyNull(objects));}
```
Comment (eugenepeh) [inline test/java/seedu/addressbook/common/UtilsTest.java:49]: Not necessary to abstract the `assertTrue(Utils.isAnyNull(...))` as separate method, because asserting the result of the method itself is fine.

Although existing test `assertAreUnique` does that, but that is to prevent `Arrays.asList(...)` from flying everywhere.

**Review by eugenepeh — COMMENTED** [review]
Good job on adding both positive and negative test cases, as well as empty and null scenario.

Some comments added. Please close the PR after reading the comments.
